### DISCUSSION TITLE
Error Rendering Component - should have parent view controller:<RNSScreen: 0x7f93ec396630> but actual parent is:<UIViewController: 0x7f93e2738ac0>

### DISCUSSION THREAD

**Discussion by hicath** [software-mansion/react-native-screens#1324] [Q&A] [answered]
Hi please kindly help, I have been stuck with this annoying problem for days and I'm totally out of ideas.
We're using `react-navigation v6` and `react-native-screens ^3.11.1`, and the error occurs when I use one of the video player library, I know it should be asked to the library author but since I have not received any answer yet, I think it can't hurt to give it a try and ask about the issue here since it's also related to RNSScreen. 

Do you have any idea what's going on here? and maybe please also provide any solution you have in mind that I can implement on the application level..

Please kindly assist, any explanation and solution would be really appreciated!

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @hicath :wave:

Information provided by you is not enough for us to help you. Sorry 😢
Currently, all we know is that you use our library and you have some kind of crash.

Please submit [an issue here](https://github.com/software-mansion/react-native-screens/issues/new/choose) with a clear description and a **reproduction** of the crash i.e. [a Snack](https://snack.expo.dev/), a snippet of code or a link to a minimal repository on GitHub that reproduces the problem. Please read [this short guide](https://stackoverflow.com/help/minimal-reproducible-example) for reference. And make sure to fill all the fields in the issue template! 😄 

Cheers
